backend/db/utils.py

```diff
@@ -170,7 +177,8 @@ def refresh_derived_tables_exec(
     t0 = datetime.now()
     for module in ddl_modules_queue:
         t0_2 = datetime.now()
-        print(f' - creating new table/view: {module}...')
+        table_or_view = 'view' if module in views else 'table'
+        print(f' - creating new {table_or_view}: {module}...')
```

Comment:
## Improve: Refresh logs
Now says 'table' if it's refreshing a table or 'view' if view instead of 'table/view'